napari/_app/constants/_menus.py

```diff
@@ -0,0 +1,30 @@
+"""All Menus that are available in the napari GUI are defined here.
+
+Internally, prefer using the `MenuId` enum instead of the string literal.
+
+SOME of these (but definitely not all) will be exposed as "contributable"
+menus for plugins to contribute commands and submenu items to.
+"""
+
+from enum import Enum
+
+
+class MenuId(str, Enum):
```

Comment:
all napari menus will live here? *love* how this feels

Reply:
yessir!  one enum to rule them all